.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
Before you submit this, make a single, separate change (PR) to add the .gitignore file and then merge that into this PR. That way you won't be including unneeded files.

You might be able to do that directly from the Github interface.